critiquebrainz/frontend/forms/review.py

```diff
@@ -32,18 +32,11 @@ class ReviewEditForm(FlaskForm):
         StateAndLength(min=MIN_REVIEW_LENGTH, max=MAX_REVIEW_LENGTH,
                        message=lazy_gettext("Text length needs to be between %(min)d and %(max)d characters.",
                                             min=MIN_REVIEW_LENGTH, max=MAX_REVIEW_LENGTH))])
-    license_choice = RadioField(
-        choices=[
-            ('CC BY-SA 3.0', lazy_gettext('Allow commercial use of this review(<a href="https://creativecommons.org/licenses/by-sa/3.0/" target="_blank">CC BY-SA 3.0 license</a>)')),  # noqa: E501
-            ('CC BY-NC-SA 3.0', lazy_gettext('Do not allow commercial use of this review, unless approved by MetaBrainz Foundation (<a href="https://creativecommons.org/licenses/by-nc-sa/3.0/" target="_blank">CC BY-NC-SA 3.0 license</a>)')),  # noqa: E501
-        ],
-        validators=[validators.InputRequired(message=lazy_gettext("You need to choose a license"))])
     remember_license = BooleanField(lazy_gettext("Remember this license choice for further preference"))
     language = SelectField(lazy_gettext("You need to accept the license agreement!"), choices=languages)
     rating = IntegerField(lazy_gettext("Rating"), widget=Input(input_type='number'), validators=[validators.Optional()])
 
-    def __init__(self, default_license_id='CC BY-SA 3.0', default_language='en', **kwargs):
-        kwargs.setdefault('license_choice', default_license_id)
+    def __init__(self, default_license_id='CC BY-SA 4.0', default_language='en', **kwargs):
```

Comment:
Since license id is gone, is default_license_id still needed?

Reply:
Ahh, I missed this. We don't need it now. I'll fix it.